ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_control_flow_operators.py

```diff
@@ -113,3 +113,41 @@ def _test_branch_2(x):
         branches=[_test_branch_1, _test_branch_2],
         operand=x[0],
     )
+
+
+@handle_frontend_test(
+    fn_tree="jax.lax.fori_loop",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        min_dim_size=1,
+    ),
+    lower=st.integers(min_value=-10, max_value=10),
+    upper=st.integers(min_value=-10, max_value=10),
+    test_with_out=st.just(False),
+)
+def test_jax_lax_fori_loop(
+    *,
+    dtype_and_x,
+    lower,
+    upper,
+    test_flags,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    def _test_body_fn(x, y):
```

Comment:
I think you can have a concatination action in your _test_body_fn, you know, adding the new int into the container.

Reply:
and I think the y can be any type, not only containers.

Reply:
Added a concatenation action here.